src/database/video_emotion_database_no_song.py

```diff
@@ -39,6 +43,16 @@ def find(self, name,
                                    ).sort([('last_updated', -1)]
                                           ).limit(1)[0]
 
+    def find_all_same_id(self, name, video_id):
+        results = []
+        data =  self._db[name].find({'video_id': video_id})
```

Comment:
You should really use "col" instead of "name" as it actually is the name of the collection and not the database name.